ComponentSnapshotTestCase/ComponentSnapshotTestCase.podspec

```diff
@@ -8,6 +8,6 @@ Pod::Spec.new do |s|
   s.platform     = :ios, '7.0'
   s.requires_arc = true
   s.source_files = '**/*.h', '**/*.m', '**/*.mm'
-  s.dependency 'FBSnapshotTestCase'
+  s.dependency 'FBSnapshotTestCase/Core'
```

Comment:
Subspec available for Swift since https://github.com/facebook/ios-snapshot-test-case/commit/5508b49217857bd3c01eef84f6015c87a333872a or thereabouts. This fails without `use_frameworks!`. Since ComponentKit is a Swift-free zone (and likely to remain so), so seems to make life a lot easier to just use the core (ObjC-only) subspec.